bin/lutris

```diff
@@ -27,16 +27,26 @@ if os.path.isdir(os.path.join(LAUNCH_PATH, "../lutris")):
 else:
     sys.path.insert(0, os.path.normpath(os.path.join(LAUNCH_PATH, "../lib/lutris")))
 
-try:
-    gettext.textdomain('lutris')
-except:
-    sys.stderr.write("Couldn't set gettext domain, translations won't work.")
-
 try:
     locale.setlocale(locale.LC_ALL, "")
 except locale.Error:
     sys.stderr.write("Unsupported locale setting. Fix your locales\n")
 
+try:
+    # Application install prefix
+    # Example:
+    #   /usr/bin/lutris/../..         -> /usr
+    #   $HOME/.local/bin/lutris/../.. -> $HOME/.local
+    prefixPath = os.path.realpath(os.path.join(__file__, os.path.pardir, os.path.pardir))
+
+    localDir = "%s/share/locale" % prefixPath
```

Comment:
I think this is pretty sketchy. It might work in all default configurations, but I'm not a fan of it. The proper way would be to configure a file via Meson, and take the localedir from there. For example like here: https://github.com/cvfosammmm/Setzer/blob/master/meson.build
Since we don't want to break the current build system just now, I think it's best to make that file "optional", i. e. design it that if it's not configured the localedir won't be set.
Also, I think `localedir` is sounds better than `localDir`, since that is what Meson uses as built-in option.

Reply:
> I'm against the autogenerated files in `po/`. This allows for example to remove an old broken translation, without deleting it for future translators.
> Also I think you should separate the `po` changes from the Chinese translation in different PRs for better diffs.

Chinese translation file removed

Reply:
> I think this is pretty sketchy. It might work in all default configurations, but I'm not a fan of it. The proper way would be to configure a file via Meson, and take the localedir from there. For example like here: https://github.com/cvfosammmm/Setzer/blob/master/meson.build
> Since we don't want to break the current build system just now, I think it's best to make that file "optional", i. e. design it that if it's not configured the localedir won't be set.
> Also, I think `localedir` is sounds better than `localDir`, since that is what Meson uses as built-in option.

I done the change. Now it will find locale dir from the optional configuration file generated by meson.

89ddec3c138ffc508145d17bdb32faee749470b3